sections/performance/nativeoverutil.md

```diff
@@ -0,0 +1,76 @@
+# Prefer native JS methods over user-land utils like Lodash
+
+
+<br/><br/>
+
+### One Paragraph Explainer
+
+Sometimes, using native methods is better than requiring `lodash` or `underscore` because it **will not lead in a performance boost** and use more space than necessary.
+The performance using native methods result in an overall ~50% gain which includes the following methods:
+ - `Array.concat`
```

Comment:
Try keeping the format like other bullets: no bold, no lists - just plain text organized with line breaks